contracts/modules/loan/BaseLoan.sol

```diff
@@ -257,7 +257,6 @@ abstract contract BaseLoan is ILoan, MutualUpgrade {
     DebtPosition memory debt = debts[positionId];
 
     uint256 totalOwed = debt.principal + debt.interestAccrued;
-    require(totalOwed == _getMaxRepayableAmount(positionId, totalOwed));
```

Comment:
yes in BaseLoan and RevolverLoanI n BulletLoan and AmoritizedLoan it will be capped at the total amount allowed for that payment period, this means they can't call `depositAndClose` until the last payment period in the loan.